packages/xstate-inspect/src/browser.ts

```diff
@@ -89,10 +89,12 @@ const getFinalOptions = (options?: Partial<InspectorOptions>) => {
 
 const patchedInterpreters = new Set<AnyInterpreter>();
 
-export function inspect(options?: InspectorOptions): Inspector | undefined {
+export function inspect(
+  options?: InspectorOptions
+): BrowserInspector | undefined {
   const { iframe, url, devTools } = getFinalOptions(options);
```

Comment:
I would move the resolving of the potential `targetWindow` option to this `getFinalOptions` call. Given the existing structure of the code, `options` should not be used anywhere in this function - except for being used as an argument for this very call here.

Reply:
Sure, I can make this change.